.github/workflows/ci.yml

```diff
@@ -93,6 +93,30 @@ jobs:
             packages/babel-standalone/*.js
             !**/node_modules/**
 
+  lint:
+    name: Lint
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Use Node.js latest
+        uses: actions/setup-node@v2-beta
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
```

Comment:
The yarn cache is actually identical on all OSes so you can have all the jobsshare the same cache

Reply:
Oh thanks! I left it with the os name because I wasn't sure about it, but I'll update now.